diy-trackers-guide.md

```diff
@@ -22,11 +22,32 @@ For more information on what these tracking options look like, please refer to t
 <iframe width="100%" height="auto" src="https://www.youtube.com/embed/Nl_6eQV32ys" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 </div>
 
+## What is an extension
+
+An extension is a singular auxiliary IMU attached to a primary tracker and placed at another location. This allows you to build a secondary tracker without the need for an extra battery, charge board and microcontroller. 
```

Comment:
I think we don't divide trackers on primary or secondary, they're all just trackers. Otherwise, it may imply that we may somehow connect the MCUs with each other (if not stated otherwise).

```suggestion
An extension is an additional IMU attached to the microcontroller and placed at another location. This allows you to build a tracker without the need for an extra battery, charge board and microcontroller.
```

Reply:
Same here, using primary tracker means it's clear it's piggy backing off the battery, switch, microcontroller etc.

Reply:
If we consider only this design, then, well, alright. 

I just feel like it's better to be in more general terms, since other non-conventional designs may be considered and we don't have this primary/secondary distinction (like, secondary tracker = auxiliary IMU, but do we really need one more term for that?)

Reply:
That's true, but right now these step by step breakdowns aren't for people who would do unconventional builds